addons/medical_treatment/XEH_preInit.sqf

```diff
@@ -35,23 +35,10 @@ GVAR(animDurations) = [] call CBA_fnc_createNamespace;
 ];
 
 // class names of medical facilities
-// global variable so it can be accessed by mission makers
-GVAR(facilityClasses) = [
-    "TK_GUE_WarfareBFieldhHospital_Base_EP1",
-    "TK_GUE_WarfareBFieldhHospital_EP1",
-    "TK_WarfareBFieldhHospital_Base_EP1",
-    "TK_WarfareBFieldhHospital_EP1",
-    "US_WarfareBFieldhHospital_Base_EP1",
-    "US_WarfareBFieldhHospital_EP1",
-    "MASH_EP1",
-    "MASH",
-    "Land_A_Hospital",
-    "CDF_WarfareBFieldhHospital",
-    "GUE_WarfareBFieldhHospital",
-    "INS_WarfareBFieldhHospital",
-    "RU_WarfareBFieldhHospital",
-    "USMC_WarfareBFieldhHospital"
-];
+GVAR(facilityClasses) = [];
+{
+    GVAR(facilityClasses) append getArray _x;
```

Comment:
```suggestion
    GVAR(facilityClasses) append ((getArray _x) select {isClass (configFile >> "CfgVehicles" >> _x)});
```

Reply:
Our checks are case sensitive, might also convert to config-case just to be safe?
Or just add to docs that it's case-sensitive and screw people who can't write correctly.